sdk/translation/azure-ai-translation-text/src/main/java/com/azure/ai/translation/text/TextTranslationAsyncClient.java

```diff
@@ -709,150 +550,131 @@ public Mono<List<TranslatedTextItem>> translate(List<String> to, List<InputTextI
+        return translateInner(Arrays.asList(targetLanguage), convertTextToData(texts));
     }
 
     /**
-     * Transliterate Text.
+     * Translate Text.
+     * <p>
+     * This method is used when you have single target language and single text to translate.
+     * </p>
      *
-     * @param language Specifies the language of the text to convert from one script to another.
-     * Possible languages are listed in the transliteration scope obtained by querying the service
-     * for its supported languages.
-     * @param fromScript Specifies the script used by the input text. Look up supported languages using the
-     * transliteration scope,
-     * to find input scripts available for the selected language.
-     * @param toScript Specifies the output script. Look up supported languages using the transliteration scope, to find
-     * output
-     * scripts available for the selected combination of input language and input script.
-     * @param requestBody Defines the content of the request.
+     * @param targetLanguage Specifies the language of the output text. The target language must be one of the
+     * supported languages included
+     * in the translation scope. For example, use to=de to translate to German.
+     * It's possible to translate to multiple languages simultaneously by repeating the parameter in the query string.
+     * For example, use to=de&amp;to=it to translate to German and Italian.
+     * @param text Defines the content of the request.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return the response body on successful completion of {@link Mono}.
+     * @return the response.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<List<TransliteratedText>> transliterate(String language, String fromScript, String toScript,
-        List<InputTextItem> requestBody) {
-        // Generated convenience method for transliterateWithResponse
-        RequestOptions requestOptions = new RequestOptions();
-        return transliterateWithResponse(language, fromScript, toScript, BinaryData.fromObject(requestBody),
-            requestOptions).flatMap(FluxUtil::toMono)
-            .map(protocolMethodData -> protocolMethodData.toObject(TYPE_REFERENCE_LIST_TRANSLITERATED_TEXT));
+    public Mono<List<TranslatedTextItem>> translate(String targetLanguage, String text) {
```

Comment:
If there's a single targetLanguage and single text, why do we return a `List<TranslatedTextItem>`?

Reply:
Fixed